components/core/src/package/archive.rs

```diff
@@ -224,7 +224,6 @@ impl PackageArchive {
         let writer = writer::Disk::new();
         let mut extract_options = ExtractOptions::new();
         extract_options.add(ExtractOption::Time);
-        extract_options.add(ExtractOption::Owner);
```

Comment:
Why was this removed?

Reply:
I actually probably never meant to extract the owner, this would preserve the user who built it. This was never a problem before because we always build software as root and run the Supervisor as root